webservices/args.py

```diff
@@ -64,37 +64,50 @@ def one_of(value, options):
+    'state': Arg(str, multiple=True, description='U.S. State candidate or territory where a candidate runs for office.'),
     'party': Arg(str, multiple=True, description='Three letter code for the party under which a candidate ran for office'),
     'year': Arg(str, dest='election_year', description='See records pertaining to a particular year.'),
     'district': Arg(str, multiple=True, description='Two digit district number'),
-    'candidate_status': Arg(str, multiple=True, description='One letter code explaining if the candidate is a present, future or past candidate'),
-    'incumbent_challenge': Arg(str, multiple=True, description='One letter code explaining if the candidate is an incumbent, a challenger, or if the seat is open.'),
+    'candidate_status': Arg(str, multiple=True, enum=['', 'C', 'F', 'N', 'P'], description='One letter code explaining if the candidate is; C = present candidate, F = future candidate, N = Not yet a candidate, P = prior candidate'),
+    'incumbent_challenge': Arg(str, multiple=True, enum=['', 'I', 'C', 'O'], description='One letter code explaining if the candidate is an incumbent, a challenger, or if the seat is open.'),
 }
 
 candidate_list = {
     'q': Arg(str, description='Text to search all fields for'),
-    'candidate_id': Arg(str, multiple=True, description="Candidate's FEC ID"),
-    'fec_id': Arg(str, description="Candidate's FEC ID"),
+    'candidate_id': Arg(str, multiple=True, description="A unique identifier assigned to each candidate registered with the FEC. If a person runs for several offices, they will have separate candidate IDs for each office."),
     'name': Arg(str, description="Candidate's name (full or partial)"),
 }
 
 committee = {
-    'year': Arg(int, multiple=True, description='A year that the committee was active- (after original registration date but before expiration date.)'),
-    'cycle': Arg(int, multiple=True, description='An election cycle that the committee was active- (after original registration date but before expiration date.)'),
-    'designation': Arg(str, multiple=True, description='The one-letter designation code of the organization'),
-    'organization_type': Arg(str, multiple=True, description='The one-letter code for the kind for organization'),
-    'committee_type': Arg(str, multiple=True, description='The one-letter type code of the organization'),
+    'year': Arg(int, multiple=True, description='A year that the committee was active- (After original registration date but before expiration date.)'),
+    'cycle': Arg(int, multiple=True, description='A 2-year election cycle that the committee was active- (after original registration date but before expiration date.)'),
+    'designation': Arg(str, multiple=True, enum=['', 'A', 'J', 'P', 'U', 'B', 'D'],
+        description='The one-letter designation code of the organization: \
```

Comment:
It wasn't rendering markdown, but now we have our own fork and I think I can just add the markdown class.